core/templates/dev/head/domain/question/EditableQuestionBackendApiService.ts

```diff
@@ -101,10 +102,50 @@ oppia.factory('EditableQuestionBackendApiService', [
       });
     };
 
+    var _addQuestionSkillLink = function(
+        questionId, skillId, successCallback, errorCallback) {
+      var addQuestionSkillLinkUrl = UrlInterpolationService.interpolateUrl(
+        QUESTION_SKILL_LINK_URL_TEMPLATE, {
+          question_id: questionId,
+          skill_id: skillId
+        });
+      $http.post(addQuestionSkillLinkUrl).then(function(response) {
+        if (successCallback) {
+          successCallback();
+        }
+      }, function(errorResponse) {
+        if (errorCallback) {
+          errorCallback(errorResponse.data);
+        }
+      });
+    };
+
+    var _addMultiQuestionSkillLink = function(
+        questionId, skillIds, successCallback, errorCallback) {
+      skillIds.forEach(function(skillId) {
+        var addQuestionSkillLinkUrl = UrlInterpolationService.interpolateUrl(
```

Comment:
Here as well, we are making multiple post requests for every skill id right?
Instead we should probably be making one single post request with a question_id and a list of skill ids (comma separated)

Reply:
Done.